webextension/background.js

```diff
@@ -129,8 +127,75 @@ const Config = (function() {
       }
     }
 
-    load() {
+    maybeSendTelemetry(message) {
+      browser.study.sendTelemetry(message);
+    }
+
+    get shieldStudySetup() {
+      const pref = `extensions.${browser.runtime.id.split("@")[0]}.variation`;
+      return {
+        activeExperimentName: browser.runtime.id,
+        allowEnroll: true,
+        studyType: "shield",
+        telemetry: {
+          send: true,
+          removeTestingFlag: !this._testingMode,
+        },
+        endings: {
```

Comment:
baseUrls  <-  plural.  Late change!    Empty array, but null is okay.

Reply:
(it's also optional so you don't need it if none!)

Reply:
Good to know, thanks. I'll be setting them for both endings that will be in the final version, so I'll use single-value arrays here.